dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/net/TracingRequestInterceptor.kt

```diff
@@ -36,7 +36,13 @@ internal class TracingRequestInterceptor(acceptedHosts: List<String> = emptyList
             if (acceptedHosts.isEmpty()) {
                 ""
             } else {
-                acceptedHosts.joinToString(separator = "|") { "^(.*\\.)*$it" }
+                acceptedHosts.joinToString(separator = "|") {
+                    if (it == "*") {
```

Comment:
shouldnt we also filter out/forbid on empty string?

Reply:
Hmmm good point...not really sure but I am waiting for the others to say their opinion also and will update it after. Tks.